lib/resources/example_json/dstu2_examples_relatedperson.rb

```diff
@@ -245,10 +245,11 @@ module Resources
               }
             ],
             "period": {
-              "start": "2015-10-30T06:48:46.000Z"
+              "start": "2015-10-30T06:50:09.000Z"
             }
           }
-        }
+        },
+        <SNIPPED>
```

Comment:
Looks like my suggestion for <SNIPPED> will not work here as I am getting a nanoc compile error.  How about an ellipsis?